src/angular-app/bellows/core/navbar.controller.ts

```diff
@@ -24,15 +24,21 @@ export class NavbarController implements angular.IController {
   displayShareButton: boolean;
   projectTypeNames: ProjectTypeNames;
   siteName: string;
+  showShareButton: boolean = false;
```

Comment:
How about we change the variable name to `isLexiconProject` and then use that variable to show/hide the settings button as well?  Right now the settings button uses a `ng-if="$ctrl.project"` which could be changed to `ng-if="isLexiconProject"`